src/assets/scss/components/_body.scss

```diff
@@ -165,8 +127,35 @@
 }
 
 
+.slider_full-slick {
+
+    .slider_full-slick-img {
+        >img {
+            width: 100%;
+        }
+    }
+
+    // .slider_fill-img {
```

Comment:
@vihan85 
không để comment code.

Reply:
Hi @huulochuynh
This issue has been resolved

Thanks best & regards